CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ breaking changes to the options supported by the plugin will be shipped as a bre
 
 ## Unreleased changes
 
+- (patch) Align code-block line numbers to the right.
```

Comment:
```suggestion
- (patch) Align code-block line numbers to the right
```
Nitpick, no period, and could this be placed after the comment (to reduce the diff for the next release)?

Reply:
Updated.